docs/index.md

```diff
@@ -9,7 +9,7 @@ title: TrackBeau
 
 * This is application that is designed for busy beauty salons with diverse customer bases that want to personalise service. While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface). <br>
   Example usages:
-  * Track customer profiles like their skin type or hair type so that appropriate products can be recommended for return customers
+  * Track customer profiles like their skinType type or hairType type so that appropriate products can be recommended for return customers
```

Comment:
Same comment as README.md and DeveloperGuide.md

Reply:
edited